wannierberri/__system.py

```diff
@@ -73,90 +68,22 @@ class System():
-    degen_thresh : float
-        threshold to consider bands as degenerate. Used in calculation of Fermi-surface integrals. Default: ``{degen_thresh}``
-    random_gauge : bool
-        applies random unitary rotations to degenerate states. Needed only for testing, to make sure that gauge covariance is preserved. Default: ``{random_gauge}``
-    ksep: int
-        separate k-point into blocks with size ksep to save memory when summing internal bands matrix. Working on gyotropic_Korb and berry_dipole. Default: ``{ksep}``
-    delta_fz:float
-        size of smearing for B matrix with frozen window, from frozen_max-delta_fz to frozen_max. Default: ``{delta_fz}``
+    _getFF : bool
+        generate the FF_R matrix based on the uIu file. May be used for only testing so far. Default : ``{_getFF}``
     use_wcc_phase: bool
         using wannier centers in Fourier transform. Correspoinding to Convention I (True), II (False) in Ref."Tight-binding formalism in the context of the PythTB package". Default: ``{use_wcc_phase}``
     wannier_centers_cart :  array-like(num_wann,3)
         use the given wannier_centers (cartesian) instead of those determined automatically. Incompatible with `wannier_centers_reduced`
     wannier_centers_reduced :  array-like(num_wann,3)
         use the given wannier_centers (reduced) instead of those determined automatically. Incompatible with `wannier_centers_cart`
 
-    """ .format(**default_parameters)
-
-    def __init__(self, old_format=False,    **parameters ):
-
-        self.set_parameters(**parameters)
-        self.old_format=old_format
-
-        cprint ("Reading from {}".format(seedname+"_HH_save.info"),'green', attrs=['bold'])
-
-        f=open(seedname+"_HH_save.info" if self.old_format else seedname+"_R.info","r")
-        l=f.readline().split()[:3]
-        self.seedname=seedname
-        self.num_wann,nRvec=int(l[0]),int(l[1])
-        self.nRvec0=nRvec
-        real_lattice=np.array([f.readline().split()[:3] for i in range(3)],dtype=float)
-        self.real_lattice,self.recip_lattice=real_recip_lattice(real_lattice=real_lattice)
-        iRvec=np.array([f.readline().split()[:4] for i in range(nRvec)],dtype=int)
-        
-        self.Ndegen=iRvec[:,3]
-        self.iRvec=iRvec[:,:3]
-
-        self.cRvec=self.iRvec.dot(self.real_lattice)
-
-        print ("Number of wannier functions:",self.num_wann)
-        print ("Number of R points:", self.nRvec)
-        print ("Recommended size of FFT grid", self.NKFFT_recommended)
-        print ("Real-space lattice:\n",self.real_lattice)
-        #print ("R - points and dege=neracies:\n",iRvec)
-        has_ws=str2bool(f.readline().split("=")[1].strip())
+    Notes:
+    -------
+        for tight-binding models it is recommended to use `use_wcc_phase = True`. In this case the external terms vanish, and 
+        one can safely use `berry=False, morb=False`, and also set `'external_terms':True` in the parameters of the calculation
```

Comment:
Should this be external terms : false? (In the sense that external terms can be neglected)

Reply:
right, corrected